tests/unit/test_dataframe.py

```diff
@@ -105,15 +105,18 @@ def test_dataframe_method_alias():
     ],
 )
 def test_copy_into_format_name_syntax(format_type, sql_simplifier_enabled):
+    def query_result(*args, **kwargs):
+        return [], [], [], None
```

Comment:
I think an integration test would be better here instead of a unit test

Reply:
I will add an integration test